### DISCUSSION TITLE
Initial Location of the Chart is not centered

### DISCUSSION THREAD

**Discussion by efecankatturkmen** [ahnaineh/org_chart#2] [General]
Firstly, I would like to tell you that I loved this implementation. But, I also have positive feedback for you. 
The chart initialized at the top left corner & I couldn't find a programmatic way to make it centered.

**Comment by efecankatturkmen** [comment]
Secondly, I can't tell why but, when I add an elevated button at the bottom of the cards, the last nodes button becomes unclickable.

**Comment by ahnaineh** [comment]
Really really sorry for this extremely late reply... 🥲
I didn't notice this before somehow (Ig life events also added up)...
I'll check this up, i also want to clean up some stuff so yeah if you have additional things don't hesitate to mention... Sorry again

**Reply by ahnaineh** [reply]
Soo version 4 is out....
And both of these were included in the update, i fixed the issue of being unable to click the nodes sometimes...
and centered it and i also exposed the method in the controller to center the graph when you want...
Check it out!
